Tools/scripts/uploader.py

```diff
@@ -771,9 +772,9 @@ def main():
                                   args.source_component)
 
                 except Exception as e:
-                    print("Exception creating uploader: %s" % str(e))
+                    #print("Exception creating uploader: %s" % str(e))
```

Comment:
why these two changes?

Reply:
I updated my initial title msg to explain

Reply:
I made this a little smarter. Loops fast on port opening error, otherwise shows the error with the same 0.05 timeout

Reply:
A 50ms delay was too long?!

Reply:
yeah, it had to loop around and try 255 serial ports!